### ISSUE TITLE
Newest published package has an error when attempting to run

### ISSUE THREAD

**Issue by mrpotatoes** [pgilad/leasot#305]
Looks like there is an issue that cropped up with the latest published package. When running the following snippet →  `$ npx leasot -x --reporter markdown './**/*.tsx' './**/*.ts' -i '**/node_modules/**/*' > TODO.md`

I receive this error
```
npx: installed 36 in 3.307s
/Users/al62643/.npm/_npx/2724/bin/leasot: line 1: import: command not found
/Users/al62643/.npm/_npx/2724/bin/leasot: line 2: import: command not found
/Users/al62643/.npm/_npx/2724/bin/leasot: line 3: import: command not found
/Users/al62643/.npm/_npx/2724/bin/leasot: line 4: import: command not found
/Users/al62643/.npm/_npx/2724/bin/leasot: line 5: import: command not found
/Users/al62643/.npm/_npx/2724/bin/leasot: line 6: syntax error near unexpected token `('
/Users/al62643/.npm/_npx/2724/bin/leasot: line 6: `const __dirname = path.dirname(fileURLToPath(import.meta.url));'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Issue breaks either as an installed package (latest) or using `npx`. I think what happened is that it wasn't transpiled back down to ES5.

* Node → v14.18.2
* npx → 6.14.15
* OS → macOS Monterey

**Comment by mrpotatoes** [comment]
When I set it to the previous package (`leasot@12.5.0`) it works fine → `npx leasot@12.5.0 -x --reporter markdown './**/*.tsx' './**/*.ts' -i '**/node_modules/**/*' > TODO.md`

**Comment by pgilad** [comment]
Thanks for reporting, it looks like the node.js hashbang isn't kept on the leasot bin files sadly. I'll try to get it fixed today.

**Comment by pgilad** [comment]
Again thanks for reporting, fixed with version `v13.1.0`. I accidentally put the shebang on the wrong files :)
